docs/source/en/model_doc/owlvit.mdx

```diff
@@ -0,0 +1,88 @@
+
+## Usage
+
+OwlViT is a zero-shot text-conditioned object detection model. OwlViT uses CLIP as its multi-modal backbone, with a ViT like transformer to get visual features and a causal language model to get the text features. To use CLIP for detection, OwlViT removes the final token pooling layer of the vision model and attaches a lightweight classification and box head to each transformer output token. Open-vocabulary classification is enabled by replacing the fixed classification layer weights with the class-name embeddings obtained from the text model. The authors first train CLIP from scratch and fine-tune it end-to-end with the classification and box heads on standard detection datasets using a bipartite matching loss. One or multiple text queries per image can be used to perform zero-shot text-conditioned object detection. 
+
+The [`CLIPFeatureExtractor`] can be used to resize (or rescale) and normalize images for the model and the [`CLIPTokenizer`] is used to encode the text. The [`OwlViTProcessor`] wraps [`CLIPFeatureExtractor`] and [`CLIPTokenizer`] into a single instance to both encode the text and prepare the images. The following example shows how to perform object detection using [`OwlViTProcessor`] and [`OwlViTForObjectDetection`].
+
+
+```python
+>>> from PIL import Image
+>>> import requests
+
+>>> from transformers import OwlViTProcessor, OwlViTForObjectDetection
+
+>>> model = OwlViTForObjectDetection.from_pretrained("adirik/owlvit-base-patch32")
+>>> processor = OwlViTProcessor.from_pretrained("adirik/owlvit-base-patch32")
+
+>>> url = "http://images.cocodataset.org/val2017/000000039769.jpg"
+>>> image = Image.open(requests.get(url, stream=True).raw)
+
+>>> inputs = processor(text=[["a photo of a cat", "a photo of a dog"]], images=image, return_tensors="pt", padding=True)
+
+>>> outputs = model(**inputs)
+>>> logits = outputs.logits 
+>>> boxes = outputs.boxes # Object box boundaries
+```
+
+This model was contributed by [adirik](https://huggingface.co/adirik). The original code can be found [here](https://github.com/google-research/scenic/tree/a41d24676f64a2158bfcd7cb79b0a87673aa875b/scenic/projects/owl_vit).
+
+## OwlViTConfig
+
+[[autodoc]] OwlViTConfig
+    - from_text_vision_configs
+
+## OwlViTTextConfig
+
+[[autodoc]] OwlViTTextConfig
+
+## OwlViTVisionConfig
+
+[[autodoc]] OwlViTVisionConfig
+
+## OwlViTProcessor
+
+[[autodoc]] OwlViTProcessor
```

Comment:
```suggestion
## OwlViTFeatureExtractor

[[autodoc]] OwlViTFeatureExtractor

## OwlViTProcessor

[[autodoc]] OwlViTProcessor
```
As there seems to be also a custom feature extractor defined, it should also be in the docs